src-ui/components/multi/MappingPane.cpp

```diff
@@ -1044,10 +1091,15 @@ struct SampleWaveform : juce::Component, HasEditor
         HZ_DRAG_LOOPEND,
     } mouseState{MouseState::NONE};
 
+    SampleCursor samplePlaybackPosition;
```

Comment:
a slightly nuanced thing. Having a component as a member means the construct and destruct time is a bit less in our control. here it is fine but it would be more idiomatic with the rest of the code to have

`std::unique_ptr<SampleCursor> samplePlaybackPosition` 

and then in the constructor below do

```
samplePlyabackPosition = std::make_unique<>();
addAndMakeVisible(*samplePlaybackPosition)
```

Reply:
ok no pb